src/basic/fileio.c

```diff
@@ -646,17 +652,15 @@ int parse_env_file(
                 newline = NEWLINE;
 
         va_start(ap, newline);
-        r = parse_env_file_internal(NULL, fname, newline, parse_env_file_push, &ap, &n_pushed);
+        r = parse_env_file_internal(NULL, fname, newline, false, parse_env_file_push, &ap, &n_pushed);
         va_end(ap);
 
         return r < 0 ? r : n_pushed;
 }
 
-static int load_env_file_push(
-                const char *filename, unsigned line,
-                const char *key, char *value,
-                void *userdata,
-                int *n_pushed) {
+static int load_env_file_push (const char *filename, unsigned line,
+                               const char *key, char *value,
+                               bool expand, void *userdata, int *n_pushed) {
```

Comment:
Please don't change that. Lennart uses this style... I'm not entirely a fan, but it does have the advantage of not requiring a reindent when function name changes and being uniform between functions. Anyway, we shouldn't gratuitously change formatting.